tests/autopilot-capacity/aut_test.go

```diff
@@ -13,13 +13,74 @@ import (
 	. "github.com/portworx/torpedo/tests"
 )
 
+var (
+	defaultCoolDownPeriod      = int64(60)
+	pxVolumeUsagePercent       = "100 * (px_volume_usage_bytes / px_volume_capacity_bytes)"
+	pxVolumeCapacityPercent    = "px_volume_capacity_bytes / 1000000000"
+	labelSelectorOpGt          = "Gt"
+	labelSelectorOpLt          = "Lt"
+	specActionName             = "openstorage.io.action.volume/resize"
+	ruleActionsScalePercentage = "scalepercentage"
+)
+
 var (
 	testNameSuite   = "AutopilotVolumeResize"
 	timeout         = 30 * time.Minute
+	scaleTimeout    = 2 * time.Hour
 	workloadTimeout = 2 * time.Hour
 	retryInterval   = 30 * time.Second
 )
 
+var Autopilotrule1 = scheduler.AutopilotRuleParameters{
+	// Resize PVC by 50% until volume usage is more than 50%
+	ActionsCoolDownPeriod: defaultCoolDownPeriod,
+	RuleConditionExpressions: []scheduler.AutopilotRuleConditionExpressions{
+		{
+			Key:      pxVolumeUsagePercent,
+			Operator: labelSelectorOpGt,
+			Values:   []string{"50"},
+		},
+	},
+	RuleActions: []scheduler.AutopilotRuleActions{
+		{
+			Name: specActionName,
+			Params: map[string]string{
+				ruleActionsScalePercentage: "50",
+			},
+		},
+	},
+	ExpectedPVCSize: 27179089920,
+}
+
+var Autopilotrule2 = scheduler.AutopilotRuleParameters{
```

Comment:
instead of `Autopilotrule2`, give a more descriptive name to the variable indicating what this combination achieves. e.g `ruleResizeIfLessThan10`